extensions/divina.md

```diff
@@ -62,6 +63,12 @@
 
 The goal of this document is to provide a profile dedicated to visual narratives for the [Readium Web Publication Manifest](https://readium.org/webpub-manifest).
 
+This profile relies on:
+
+* the use of [presentation hints](./presentation.md) for specifying display constraints, 
+* the definition of a new collection type for implementing guided navigation,
```

Comment:
A link to the proper section in the document would be useful here.

Reply:
interesting. Done.